scio-test/src/test/scala/com/spotify/scio/ScioResultTest.scala

```diff
@@ -30,4 +29,20 @@ class ScioResultTest extends PipelineSpec {
     r.state shouldBe State.DONE
   }
 
+  it should "not support retrieval of accumulators during template creation" in {
+    val sc = ScioContext.forTest()
```

Comment:
It's just so that I can use the `maxI` accumulator in the assertion. With `runWithContext` the declaration is locked into the inner scope, but the `ScioResult` is accessed in the outer scope where `maxI` is not defined.